usergroup/owasp.md

```diff
@@ -1,20 +1,12 @@
-# OWASP Stammtisch Frankfurt (OWASP)
-![OWASP Stammtisch Frankfurt](./owasp.logo.png)
+# IT-Security Stammtisch Frankfurt (OWASP, u.w.)
+![IT-Security Stammtisch Frankfurt](./owasp.logo.png)
 
-OWASP ist eine unabhängige, weltweite Community mit dem Ziel, die Bedeutung der Sicherheit von Webanwendungen "sichtbar zu machen". Know-How zur Entwicklung und den Betrieb sicherer Webanwendungen zu verbreiten und frei zur Verfugung zu stellen. Wir treffen uns jeden letzten Donnerstag im Monat um 19:30 bei einem wechselnden Gastgeber. Üblicherweise gibt es mindestens einen Vortrag mit Erfahrungsaustausch. Danach begeben wir uns zum geselligen Ausklang in eine Lokalität in der Nachbarschaft.
+OWASP ist eine unabhängige, weltweite Community mit dem Ziel, die Bedeutung der Sicherheit von Webanwendungen "sichtbar zu machen", Know-How zur Entwicklung und den Betrieb sicherer Webanwendungen zu verbreiten und frei zur Verfügung zu stellen. Wir treffen uns alle zwei Monate am letzten Mittwoch im Monat um 19:30 bei einem wechselnden Gastgeber. Üblicherweise gibt es mindestens einen Vortrag mit Erfahrungsaustausch. Danach begeben wir uns zum geselligen Ausklang in eine Lokalität in der Nachbarschaft.
 
 ## Links &amp; Kontakt
 
-Homepage: <https://www.owasp.org/index.php/OWASP_German_Chapter_Stammtisch_Initiative/Frankfurt>
-
-
-
-Twitter: [@OWASP_f](https://twitter.com/@OWASP_f)
-
-
-
-
-
-Mailingliste: <https://lists.owasp.org/mailman/listinfo/owasp-frankfurt>
+* Meetup: <http://www.meetup.com/de/IT-Security-Stammtisch-Frankfurt-OWASP-u-w/>
+* Homepage: <https://www.owasp.org/index.php/OWASP_German_Chapter_Stammtisch_Initiative/Frankfurt>
```

Comment:
Link ist ein 404.

Dieser scheint korrekt zu sein: https://owasp.org/www-chapter-germany/stammtische/frankfurt/

Reply:
Ups, ja, da hing der Commit zu lange herum, gab in der Zwischenzeit einen OWASP Relaunch und eine Mailinglistenmigration, habe mal beide Links angepasst. Danke fürs Prüfen :)